app/_components/Home/Sponsor/index.jsx

```diff
@@ -1,9 +1,29 @@
-// TODO: Create a Sponsor component.
-export default function Sponsor() {
-    return (
-      <main>
-        <h1 id="sponsor" className="text-6xl font-bold">Sponsor</h1>
-      </main>
-    );
-  }
-  
\ No newline at end of file
+import Image from "next/image";
+import sponsorData from "../../../data/home/sponsors.json";
+//Use Transparent sponsor logos for best effect
+const Sponsors = () => {
+  return (
+    <div className="text-center justify-center py-8  ">
```

Comment:
Add `flex` to the class list to make `justify-center` effective, as it requires a flex container, or remove the `justify-center` as it isn't affecting the output.